sagemaker_model_monitor/introduction/SageMaker-ModelMonitoring.ipynb

```diff
@@ -637,11 +646,18 @@
     "### Violations report"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
```

Comment:
This seems like empty cell

Reply:
@ushisays Can you review and revise the affected cell?

Reply:
Removed with a new commit